modules/ecr/locals.tf

```diff
@@ -0,0 +1,62 @@
+locals {
+  # Sets max amount of the latest develop images to be kept
+  image_limit = 5
+
+  # The namespace for the ECR
+  namespace = "my-ecr"
```

Comment:
I understand that it is IC, and it is not a client project, but let's avoid `my-` and similar namings, because we are not using them in client projects. It could have project name prefix `devops-ic-ecr` for example

Reply:
Thanks for your feedback, it was updated in d245cdc ✅